### DISCUSSION TITLE
questions on licence / privacy

### DISCUSSION THREAD

**Discussion by jwaagen** [mozilla/hubs#3901] [General Questions & Discussion] [answered]
Hello,

We are considering creating VR presentations in Mozilla Hubs with archaeological data that may not be further distributed, and we can run into legal issues if it does. I have studied these:

https://github.com/mozilla/hubs/blob/master/TERMS.md
https://github.com/mozilla/hubs/blob/master/PRIVACY.md

It seems that Mozilla Hubs only uses the uploaded content for the operation of hubs and only for promotion when opting for the CC-BY 3.0 licence, but could you confirm this? Are there any legal implications if you upload 3D data to Mozilla Hubs that I should be aware of in this case?

Are there in this regard differences between the terms/privacy policies for Hubs Cloud?  

In addition, I would like to understand whether it would already be possible to host our own Mozilla Hubs environment? 

Best,

Jitte Waagen

**Comment by misslivirose** [comment, marked as answer]
Hey @jwaagen - I can answer some of your questions, but can't provide legal guidance around the terms of service. 

The terms for operating Hubs Cloud are different than for hubs.mozilla.com, because Hubs Cloud is deployed software vs. a service that we run ourselves, and can be found [here](https://d7umqicpi7263.cloudfront.net/eula/product/5f452d36-bfff-4c02-a417-5c663b64fd08/64589884-fe84-45f7-b1a2-64db8b157b3f.pdf). Hubs Cloud admins can specify their own privacy policies and terms of service through the admin console. 

For hosting your own environment, the supported route is through the Hubs Cloud AWS marketplace, though there are a few other [threads ](https://github.com/mozilla/hubs/discussions/2699)where people have discussed potential alternate hosting solutions.

**Reply by jwaagen** [reply]
Thanks you @misslivirose, I am checking out clouds at the moment. 

Would there be anyone else that could answer the first part of my question, about the legal implications of uploading 3D content?

**Reply by misslivirose** [reply]
Our team can't give legal advice - we recommend consulting with your own council for questions that you have related to how the license and terms may be interpreted for your specific needs and use case for Hubs.
